djangocms_pageadmin/monkeypatch.py

```diff
@@ -40,3 +42,32 @@ def inner(self, **kwargs):
 
 
 VersioningToolbar._add_view_published_button = new_view_published_button(VersioningToolbar._add_view_published_button)
+
+
+def published_date(self, obj):
+    version = StateTracking.objects.filter(version_id=obj.pk)
+    if hasattr(version.first(), "new_state") and version.first().new_state == "published":
+        return version.first().date
+    return ""
+
+
+published_date.short_description = "Published Date"
+admin.VersionAdmin.published_date = published_date
+
+
+def get_list_display(func):
+    """
+    Register the published date field with the Versioning Admin
+    """
+    def inner(self, request):
+        original_list_display = func(self, request)
+        list_display_list = list(original_list_display)
+        # Removing created date from versioning changelist
+        del list_display_list[1]
```

Comment:
This is reliant on `created` always being the second item in the list, so if this changed it would require an update to your code here as well.

A safer way would be to use `list.index()` a bit like you did further down, and adding in error handling to avoid throwing a 500 error. You could then use that index to change the list in place and return it. Something like:
```
def inner(self, request):
    original_list_display = func(self, request)
    list_display_list = list(original_list_display)
    
    try:
        created_index = list_display_list.index("created")
    except ValueError:
        # if created is missing return the original list to avoid an error
        return original_list_display

    # change the value at the correct index
    list_display_list[created_index] = "published_date"
    # get_list_display accepts a list so can be returned directly
    return list_display_list
```

Reply:
Thanks